experimental/adaptive-dialog/README.md

```diff
@@ -44,6 +44,14 @@ You can report any issues you find or feature suggestions on our GitHub reposito
 You can use this [Visual studio code debugger extension][18] to debug both code based as well as declaratively defined Adaptive Dialogs.
 
 ## Change Log
+### 4.7 PREVIEW
+- \[**New\] Adaptive dialogs have been merged into `botbuilder-dotnet` master branch and now is built on top of the core SDK.
+- \[**New\] ***BREAKING CHANGE*** Renamed `Steps` -> `Actions`, `Rules` -> `Triggers`. Updated all triggers to follow `OnXXX` notation. 
+- \[**New\] Updating Adaptive dialogs to drop `Steps` tied directly to the adaptive dialog. If you need to run a set of actions when a dialog begins, add them to the `OnBeginDialog` trigger.
+- \[**New\] Updated and streamlined properties for actions.
```

Comment:
without context - it is not clear what does it mean to streamline properties for actions.

Reply:
Reworded and added additional context.